eng/testing/tests.wasm.targets

```diff
@@ -27,7 +27,7 @@
     <UseSystemResourceKeys>true</UseSystemResourceKeys>
     <EnableUnsafeUTF7Encoding>false</EnableUnsafeUTF7Encoding>
     <HttpActivityPropagationSupport>false</HttpActivityPropagationSupport>
-    <DebuggerSupport>false</DebuggerSupport>
+    <DebuggerSupport>true</DebuggerSupport>
```

Comment:
I wonder if we should really change the default here since we want to match what Blazor sets by default, i.e. `UseDefaultBlazorWASMFeatureSwitches`.

Reply:
actually, looks like Blazor defaults it to true when configuration is debug and provides an override:
https://github.com/dotnet/sdk/blob/1e4913f32344ce69ee17229168906140e8f88f0d/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets#L59

maybe we should do the same?

Reply:
What do we use `UseDefaultBlazorWASMFeatureSwitches` for?

`<DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(Configuration)' != 'Debug'">false</DebuggerSupport>`

Hm this would mean that a dev would need to explicitly set `/p:DebuggerSupport=true` to get the line numbers, when building in `Release`. I was hoping for line numbers in test runs too.

Reply:
`eng/testing/tests.mobile.targets:    <UseDefaultBlazorWASMFeatureSwitches Condition="'$(UseDefaultBlazorWASMFeatureSwitches)' == ''">true</UseDefaultBlazorWASMFeatureSwitches>`

This defaults to true, and I don't see it being set in anywhere in our repo. So, is the intention to always test with this?
The question is whether it is OK to use `DebuggerSupport=true` by default then?

Reply:
I don't know the history of `UseDefaultBlazorWASMFeatureSwitches` but presumably we wanted to actually test the same configuration that Blazor will use in the end.

I'd love to have line numbers by default but I don't know if that changes runtime behavior in a way that makes it risky, maybe ask Zoltan/Aleksey.

If we decide to do it I think we should move this property out of the `UseDefaultBlazorWASMFeatureSwitches` propertygroup since it won't be matching Blazor anymore and cause confusion.